app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionTab.kt

```diff
@@ -0,0 +1,14 @@
+package io.github.droidkaigi.confsched2018.presentation.sessions
+
+import io.github.droidkaigi.confsched2018.model.SessionSchedule
+import io.github.droidkaigi.confsched2018.model.Room as RoomModel
+
+sealed class SessionTab {
+    data class Room(val stuffs: List<RoomModel>) : SessionTab()
+    data class Schedule(val stuffs: List<SessionSchedule>) : SessionTab()
+}
+
+sealed class SessionTabMode {
```

Comment:
Can you use enum? 👀

Reply:
It's okay for me though, can I ask the reason? just in case.

Reply:
Enum is simple
Example:
* `is` is needed when using `when`
* Enum not need ` : SessionTabMode()`

Reply:
In this case, subclasses of SessionTabMode are defined as `object`, then the instance equality is assured. Because of that, case-statements doesn't require `is`.

However, it relied on define-style of subclasses. It might be a flaw 😇 
So, I'd love to agree on your suggestion. Thanks!

Reply:
I don't know that 🙇

Reply:
Please use upper case if you use enum 🙇 
https://android.github.io/kotlin-guides/style.html#enum-classes